modules/debug.py

```diff
@@ -60,7 +60,7 @@ def logfile(all=False):
     if all:
         title = 'Last 100 lines from the system log (/var/log/syslog)'
         cmd = 'tail -n 100 /var/log/syslog'
-    lines = subprocess.check_output(cmd, shell=True)
+    lines = subprocess.check_output(cmd, shell=True).decode("utf-8")
```

Comment:
Better to use debug.subprocess_check_output since it does decode already and it puts it in a central location

Reply:
That works fine for me

Reply:
I experimented with using **kwargs, and also just adding a shell default that could be overridden, but both seemed to break other calls - notably the tvservice calls.   Since you gave me the out, I decided that it was probably better to leave this alone for now, and return once the bugs from moving to python3 were in a better shape.